vignettes/git_usage.Rmd

```diff
@@ -116,8 +116,8 @@ Once the review is completed, the reviewer will merge the Pull Request and the
 feature branch will automatically be deleted.
 
 After merging the Pull Request the corresponding issue must be moved to the
-"Sprint Done" column on the "admiral scrum board" by the developer. The issue
-must not be closed. It will be closed automatically once the `devel` branch is
+"Sprint Done" column on the "admiral scrum board" by the developer. **The issue
+must not be closed**. It will be closed automatically once the `devel` branch is
```

Comment:
Possible to add a Screen shot of where this is on the Issue?

![image](https://user-images.githubusercontent.com/10111024/155591036-9aad444f-2799-47f6-a808-19102a5d4e6b.png)

Reply:
okay i will make get a screen shot.

Reply:
Got it! Will try my best to phrase it right.